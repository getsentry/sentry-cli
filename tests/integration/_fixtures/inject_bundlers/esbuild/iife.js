(()=>{function t(r,o){r(o)}function n(r,o){t(r,o)}function f(r,o){n(r,o)}f(function(o){throw new Error(o)},"boop");})();
//# sourceMappingURL=iife.js.map
