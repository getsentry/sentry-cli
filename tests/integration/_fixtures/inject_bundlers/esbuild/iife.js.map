{
  "version": 3,
  "sources": ["../src/bar.js", "../src/foo.js", "../src/index.js"],
  "sourcesContent": ["export function bar(fn, msg) {\n  fn(msg);\n}\n", "import { bar } from './bar.js';\n\nexport function foo(fn, msg) {\n  bar(fn, msg);\n}\n", "import { foo } from './foo.js';\n\nfunction wat(fn, msg) {\n  foo(fn, msg);\n}\n\nwat(function hello(msg) {\n  throw new Error(msg);\n}, 'boop');\n"],
  "mappings": "MAAO,SAASA,EAAIC,EAAIC,EAAK,CAC3BD,EAAGC,CAAG,CACR,CCAO,SAASC,EAAIC,EAAIC,EAAK,CAC3BC,EAAIF,EAAIC,CAAG,CACb,CCFA,SAASE,EAAIC,EAAIC,EAAK,CACpBC,EAAIF,EAAIC,CAAG,CACb,CAEAF,EAAI,SAAeE,EAAK,CACtB,MAAM,IAAI,MAAMA,CAAG,CACrB,EAAG,MAAM",
  "names": ["bar", "fn", "msg", "foo", "fn", "msg", "bar", "wat", "fn", "msg", "foo"]
}
