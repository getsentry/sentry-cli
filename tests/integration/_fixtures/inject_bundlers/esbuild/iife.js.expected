
!function(){try{var e="undefined"!=typeof window?window:"undefined"!=typeof global?global:"undefined"!=typeof self?self:{},n=(new Error).stack;n&&(e._sentryDebugIds=e._sentryDebugIds||{},e._sentryDebugIds[n]="dd751360-5eae-531d-87f4-5e00c846a9a3")}catch(e){}}();
(()=>{function t(r,o){r(o)}function n(r,o){t(r,o)}function f(r,o){n(r,o)}f(function(o){throw new Error(o)},"boop");})();
//# sourceMappingURL=iife.js.map

//# debugId=dd751360-5eae-531d-87f4-5e00c846a9a3
